contracts/tokenwrapper/src/msg.rs

```diff
@@ -0,0 +1,27 @@
+use schemars::JsonSchema;
+use serde::{Deserialize, Serialize};
+
+#[derive(Serialize, Deserialize, Clone, Debug, PartialEq, JsonSchema)]
+pub struct InstantiateMsg {
+    pub count: i32,
+}
+
+#[derive(Serialize, Deserialize, Clone, Debug, PartialEq, JsonSchema)]
+#[serde(rename_all = "snake_case")]
+pub enum ExecuteMsg {
```

Comment:
What are these messages for?

Reply:
My bad.
Removed in https://github.com/webb-tools/protocol-cosmwasm/pull/30/commits/6544b7dbefc1a0673d4588b43569cf5ec73ae239